src/rules/check-domains-against-connection-aliases.js

```diff
@@ -5,10 +5,11 @@
  * 
  * Check user email domain matches domains configured in connection
  * 
- * This rule will check that the email the user has used to login matches any of the domains configured in a connection. If there are no domains configured, it will allow access.
+ * This rule checks if the user's login email matches any domains configured in a connection. If there are no matches, the login is denied. But, if there are no domains configured it will allow access.
```

Comment:
we can say "enterprise connection" instead of just "connection", as `options.tenant_domain` is the domain being used for authentication in an **Enterprise** connection